modules/genomic_browser/test/genomic_browserTest.php

```diff
@@ -236,6 +225,7 @@ function testGenomicBrowserFilterEachTab()
     function _testPageUIs($url,$ui)
     {
         $this->safeGet($this->url . $url);
+        sleep(20);
```

Comment:
This sleep(20) need to be removed. Sorry, I forget to delete this line.

Reply:
@driusan @kongtiaowang done!